script/statesync.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+# microtick and bitcanna contributed significantly here.
+
+
+INTERVAL=1000
+
+# GET TRUST HASH AND TRUST HEIGHT
+
+LATEST_HEIGHT=$(curl -s 162.55.132.230:2021/block | jq -r .result.block.header.height);
+BLOCK_HEIGHT=$(($LATEST_HEIGHT-$INTERVAL)) 
+TRUST_HASH=$(curl -s "162.55.132.230:2021/block?height=$BLOCK_HEIGHT" | jq -r .result.block_id.hash)
```

Comment:
Using at least a variable for the node would be preferred.  Who runs this node?  It's not great if the build fails because of some random node going down.

Reply:
it's me, and my hope is for others to add their own nodes, and at that point, we pick e'm from a list.